internal/tfimport/tfimport.go

```diff
@@ -761,8 +761,10 @@ func planAndImport(rn, importRn runner.Runner, runArgs *RunArgs, skipped map[str
 		// err will be `exit code 1` even when it failed because the resource is not importable or already exists.
 		case NotImportable(output):
 			log.Printf("Import not supported by provider for resource %q\n", cc.Address)
+			skipped[cc.Address] = true
```

Comment:
With this change, these resources will be listed under `Skipped` instead of `Failed to import`?

Reply:
Currently they're not listed at all.